--- gdbinit.py	2023-09-24 13:35:54.357104009 +0800
+++ gdbinit_modified.py	2023-09-24 13:35:49.257104007 +0800
@@ -20,29 +20,6 @@
 directory = path.expanduser(directory)
 directory = path.abspath(directory)
 
-# Get virtualenv's site-packages path
-venv_path = os.environ.get("PWNDBG_VENV_PATH")
-if not venv_path:
-    venv_path = os.path.join(directory, ".venv")
-
-if not os.path.exists(venv_path):
-    print(f"Cannot find Pwndbg virtualenv directory: {venv_path}: please re-run setup.sh")
-    sys.exit(1)
-
-site_pkgs_path = glob(os.path.join(venv_path, "lib/*/site-packages"))[0]
-
-# add virtualenv's site-packages to sys.path and run .pth files
-site.addsitedir(site_pkgs_path)
-
-# remove existing, system-level site-packages from sys.path
-for site_packages in site.getsitepackages():
-    if site_packages in sys.path:
-        sys.path.remove(site_packages)
-
-# Set virtualenv's bin path (needed for utility tools like ropper, pwntools etc)
-bin_path = os.path.join(venv_path, "bin")
-os.environ["PATH"] = bin_path + os.pathsep + os.environ.get("PATH")
-
 # Add gdb-pt-dump directory to sys.path so it can be imported
 gdbpt = path.join(directory, "gdb-pt-dump")
 sys.path.append(directory)
