--- pwndbg-2024.02.14/pwndbg/gdblib/vmmap.py	2024-02-15 02:42:54.000000000 +0800
+++ vmmap.py	2024-05-24 07:49:58.989924168 +0800
@@ -96,10 +96,7 @@
     if is_corefile():
         return tuple(coredump_maps())
 
-    proc_maps = None
-    if pwndbg.gdblib.qemu.is_qemu_usermode():
-        # On Qemu < 8.1 info proc maps are not supported. In that case we callback on proc_tid_maps
-        proc_maps = info_proc_maps()
+    proc_maps = info_proc_maps()
 
     if not proc_maps:
         proc_maps = proc_tid_maps()
